config/rbac/metrics_auth_role.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: proxy-role
+  name: metrics-auth-role
```

Comment:
shouldn't this have also labels?
    app.kubernetes.io/name: build-service
    app.kubernetes.io/managed-by: kustomize

Reply:
those are auto-generated by operator-sdk